Makefile

```diff
@@ -19,10 +19,16 @@ clean:
 clean_all:
 	$(MAKE) clean
 	$(MAKE) clean_node_modules
+	$(MAKE) clean_dist
 
 clean_node_modules:
 	rm -rf node_modules
 	rm -rf packages/*/*/node_modules
+	rm -rf examples/*/node_modules
+
+clean_dist:
+	rm -rf packages/*/*/dist
+	rm -rf packages/{react-aria,react-aria-components,react-stately}/dist
```

Comment:
any thoughts about using `rm -rf packages/**/dist`? The double star matches an arbitrary number of levels we don't have to be super specific here

Reply:
I think we shouldn't do that, it'd match `packages/@react-spectrum/modal/node_modules` for example, and this is a function to wipe out dist, not node_modules. I use them independently some times

Reply:
For my own knowledge, is there a scenario where a package within our top level packages has its own `node_modules`? Mainly wondering for the verdaccio script since it currently does the double star pattern to remove dists.

Reply:
you mean `react-aria` and `react-stately`? they shouldn't have their own node_modules ever, since they only combine our lower level packages. Unless they started to depend on specific packages none of the lower levels use, then we won't see it, and I think that's highly unlikely.